vscode-dotnet-runtime-library/src/Acquisition/RedHatDistroSDKProvider.ts

```diff
@@ -0,0 +1,66 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import { diff } from 'semver';
+import { GenericDistroSDKProvider } from './GenericDistroSDKProvider';
+import { DotnetDistroSupportStatus } from './LinuxVersionResolver';
+import * as path from 'path';
+import { version } from 'os';
+/* tslint:disable:no-any */
+
+export class RedHatDistroSDKProvider extends GenericDistroSDKProvider {
+    private getCloestDistroVersion(distroVersions: any, versionTarget: number, versionKey: string)
```

Comment:
It would be helpful to have a docstring here describing what this function does and most importantly why we want to have the closest distro version (I think I see what you are doing here: rounding up or down the RHEL version) which is a good idea to save space on the json file!